_posts/16-10-01-Books.md

```diff
@@ -19,6 +19,7 @@ for modern, secure, and fast cryptography.
 
 ### Paid Books
 
+* [PHP&MYSQL](https://phpandmysql.com/) - The updated version of classic PHP book by Jon Duckett with amazing illustrations that teaches how to build dynamic websites with PHP and MYSQL.
```

Comment:
"The updated version of classic PHP book" is missing a "the".

I am unfamiliar with its history, so why is the book a "classic", and why is it relevant to mention now that it is an "updated version"?

Reply:
I might be overexcited about this book since I like it so much and believe it deserves a place on the list. Here is a more neutral description:
```
PHP book with amazing illustrations that covers all the fundamentals of PHP and MYSQL with practical examples. 
```

Reply:
Your excitement is forgiven. :smile: If you'd replace 'amazing' with 'excellent', it works for me.

Reply:
Done

Reply:
Fine.

Just noticed that four later entries in that bullet list, from Building Secure PHP Apps through Scaling PHP, do not end in a period (.). Could you add them? Saves a separate PR.

Reply:
I added periods